app/services/permitted_attributes/business_address.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module PermittedAttributes
+  class BusinessAddress
```

Comment:
Great :+1: I think we can delete `:company` from the _other_ version of this class as well (`PermittedAttributes::Address`).